config/initializers/informed.rb

```diff
@@ -0,0 +1 @@
+Informed.logger = Rails.logger
```

Comment:
thoughts on putting this logger stuff in a separate PR? feels separate from this admin feature to me

Reply:
We have a card "up next" for logging -- I could tackle that and use the work you have here w/r/t logging https://trello.com/c/WqpFBnYO/179-develop-strategy-for-logging

Reply:
@jessieay Go for it! I was using it mostly for debugging and am happy to rip it out.